packages/slate-editor/src/extensions/press-contacts/PressContactsExtension.tsx

```diff
@@ -24,6 +24,13 @@ export const PressContactsExtension = (): Extension => ({
     },
     isElementEqual(element, another) {
         if (isContactNode(element) && isContactNode(another)) {
+            if (
+                !isEqual(element.layout, another.layout) ||
+                !isEqual(element.show_avatar, another.show_avatar)
+            ) {
+                return false;
+            }
```

Comment:
Why use `isEqual` for primitive values?

```suggestion
            if (element.layout !== another.layout || element.show_avatar !== another.show_avatar) {
                return false;
            }
```

Reply:
Ah, because at first I only did one `isEqual` call with two objects containing both of these properties but it didn't look nice when formatted so I've split it into two calls but didn't realize at that point it's not needed 🙈 Changed it.

Thanks!